crates/typos/src/tokens.rs

```diff
@@ -264,13 +264,16 @@ mod parser {
         <T as nom::InputIter>::Item: AsChar + Copy,
     {
         // Size considerations:
-        // - sha-1 is git's original hash
-        // - sha-256 is git's new hash
-        // - Git hashes can be abbreviated but we need a good abbreviation that won't be mistaken
-        //   for a variable name
-        const SHA_1_MAX: usize = 40;
-        const SHA_256_MAX: usize = 64;
```

Comment:
Please carry over the comments as they are also important context to carry forward,. though without the `const`s, you'll need to add to them the sizes for each hash

Reply:
Thanks!  Go ahead and fixup and re-push and we'll get this merged

(I wish Github had native support for `fixup`)